src/Morphic-Core/Morph.class.st

```diff
@@ -2881,7 +2880,7 @@ Morph >> handleMouseDown: anEvent [
 	anEvent hand newMouseFocus: self event: anEvent.
 
 	"Check to open halo"
-	anEvent isSpecialGesture
+	(anEvent isSpecialGesture and: [self cmdGesturesEnabled])
```

Comment:
Hah! good catch!

I wonder why do we have that method in click event, but that can be for another time...